### ISSUE TITLE
[BUG] Conventional markdown images and links not rendered correctly

### ISSUE THREAD

**Issue by nekr0z** [dillonzq/LoveIt#379]
A Hugo user tries LoveIt for his/her existing site. The site has a lot of conventional Markdown like this:

```
[![image alt text](featured.jpg "hover text")](https://some-great.site/some-page.html)
```

This is standart Markdown that works in almost any Hugo theme. It makes use of Page Bundles (`featured.jpg` is in the same subdirectory and `index.md`).

The result is supposed to be the image of `featured.jpg` with alt text "image alt text". The image is a link to `https://some-great.site/some-page.html`, and when the mouse pointer is hovered over it "hover text" is displayed as tooltip.

### Expected behavior

Image renders as described.

### Actual behavior

Image does not render at all. Link to the external site is lost. The user needs to fix all his/her existing pages from last 5 years.

**Comment by nekr0z** [comment]
Still an issue, please reopen!

## this code:
```
[![image alt text](featured.jpg "hover text")](https://some-great.site/some-page.html)
```

## should render:
`featured.jpg` with `image alt text` as Alt text and `hover text` as Title, and make it a link to `https://some-great.site/some-page.html`

## actual behaviour:
figure preview of `featured.jpg` with `image alt text` as Title and `hover text` as figure caption; figure is a link to `featured.jpg`, link to `https://some-great.site/some-page.html` is lost.

**Comment by py563** [comment]
I followed #191, #375 and #388. I read about page bundles in hugo website. Is Content bundles similar to having new blog post as `newblogpost.md` directly under `content/posts` or including images from static folder or assests folder termed as content bundle? including a image from static folder in featured-image doesn't even create that div class let alone displaying image.

**Comment by nekr0z** [comment]
For a Content Bundle you create a `this-is-a-post` (or any) directory under `content/posts`, and put all your files for the post inside it. The fronmatter and text go into `this-is-a-post/index.md`, and the image goes in `this-is-a-post/picture.jpeg`. You simply refer the image in the post and frontmatter as `picture.jpeg` - no need to give the path.

So if you have a `nice_picture.png` in your `this-is-a-post` directory, you can put
```
featuredImage = "nice_picture.png"
```
in the FrontMatter section of `this-is-a-post/index.md`, and this image becomes the featured image for that post. I tested, it works.

**Comment by py563** [comment]
i thought that is called page bundle as per this [hugo article](https://gohugo.io/content-management/page-resources/) and it works fine as used in theme example.

**Comment by nekr0z** [comment]
> i thought that is called page bundle as per this [hugo article](https://gohugo.io/content-management/page-resources/) and it works fine as used in theme example.

Yes, you're right, I confused the terms. Anyway, this is not what this bug report is about.

**Comment by py563** [comment]
I know, as all the other tickets were closed and this seemed a remotely related ticket, i commented here rather than creating a new issue.
